src/main/java/com/rabbitmq/perf/MulticastParams.java

```diff
@@ -28,6 +28,8 @@
     private long confirm = -1;
     private int consumerCount = 1;
     private int producerCount = 1;
+    private int consumerChannels = 1;
```

Comment:
Agreed with Simon to call this `consumerChannelsPerConnection`. Would you prefer we used `consumerChannelsPerConnectionCount`?